testing/btest/scripts/base/protocols/rdp/rdp-rdpeudp-handshake-success.zeek

```diff
@@ -0,0 +1,4 @@
+# @TEST-EXEC: zeek -r $TRACES/rdp/rdp-rdpeudp-handshake-success.pcap %INPUT
+# @TEST-EXEC: btest-diff conn.log
```

Comment:
Diffing the `conn.log` is ok to verify the `service` is getting added, but I'd extend these tests a bit to include handlers for the various events you added.  e.g. in a `rdpeudp_established` handler, have it print out relevant info (event name + argument values) and then also have the test `btest-diff .stdout`

Reply:
Ah yes! That makes sense. I've added those.